_posts/2019-07-03-reinforce-2019-recap.md

```diff
@@ -0,0 +1,104 @@
+
+The event was held in Boston Convention & Exhibition Center, which is a giant expo hall with many levels. In the middle of the hall, you had a "Security Learning Hub" which was basically an exhibition area for partners. Despite the fact that you didn’t need to change venues like in re:Invent, you still had to walk quite a lot during the days and getting from one end of the convention center to the other took some time.
+
+![Security Learning Hub](/img/reinforce-2019-recap/security-learning-hub.jpg)
+
+If you had been to re:Invent before, everything seemed quite familiar. Unfortunately this also applied to the registration process for the sessions. Before the reserved seating registration opened, you had no way to view the sessions you were interested in a calendar view. Luckily [the same handy tool someone had made for re:Invent planning](http://reinvent-planner.aws.carlosesilva.com) also worked for re:Inforce with slight modifications, and because you only had 200 sessions to choose from instead of 2000, fitting the sessions into your calendar was also much easier. Also, you didn't need to worry about if you had enough time to travel between venues during the day, which was a huge plus as now you could actually select the sessions from everything available instead of just the ones located in the venue your previous or next session was going to be, like in re:Invent. Once the registration and reserved seating was done, everything on-site worked smoothly and everything was well organized, just like in re:Invent.
+
+ All the different breakout session types were also similar to the ones in re:Invent. Regular sessions were an hour long talks with a short Q&A in the end, chalk talks were shorter talks with more time dedicated to Q&A or a demo, workshops were 2 hour long hands-on labs where you could solve problems on your own or as a group in a table, and builders sessions were an hour long hands-on "mini-workshops" with just 5 other people and an AWS expert. This time, I wanted to focus on hands on learning by going into many workshops, builders sessions and also to the four hour long security jam hackathon on the second day.
+
+Here are some short recaps on the different sessions that I attended.
+
+# Day 1
+
+## [Keynote by Stephen E. Schmidt, VP and CISO of AWS](https://www.youtube.com/watch?v=FKphJNfpWk8)
+
+Not that many new announcements were made during the keynote, as the biggest announcements (Security Hub general availability) were already published before the keynote. It was still a decent overview of the current state of security in the cloud, how it has evolved and how it will evolve in the future (more automation and machine learning). The customer stories of big banks and insurance companies going all in on cloud and building and releasing their own security related tools and products were also a bit surprising to hear.
```

Comment:
```suggestion
Not that many new announcements were made during the keynote, as the biggest announcements (Security Hub general availability) were already published beforehand. It was still a decent overview of the current state of security in the cloud, how it has evolved and how it will evolve in the future (more automation and machine learning). The customer stories of big banks and insurance companies going all in on cloud and building and releasing their own security related tools and products were also a bit surprising to hear.
```